src/containers/List/ListFilters/BooleanFilter.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
```

Comment:
Suggestion to add these to a `components/List` folder since it seems to be more visualisation done here - not much logic involved

Reply:
Hmm, I've been pretty placid on the folder structure front, but I really think it's more important to group components/container by functional areas rather then by differentiation of how much logic they contain. I am wondering if I am missing something, what is the benefit of differentiating between components and containers ?

Reply:
I think the components should be something that can be more easily reused if they aren't allocated inside one logic-area folder. But since our components some times aren't following the strict of only doing visualisation that might not be the case. I'm just pointing out that this is in the wrong folder with the current folders structure we have.

Reply:
To understand more about the container and component pattern design se here: 
That's not a rule, and if we disagree to use it, I would rather move things to the correct folder, so it makes sense to everyone than keeping as it without using the proper sorting of files.

Reply:
Good point, it's definitely not consistent with existing projects structure, so far we have deviations in lookup tables, filter and template builder. I feel like re-structuring the whole project would be a big task, was thinking, maybe it's ok to have mix of different structure styles ?